src/ebuckets.h

```diff
@@ -275,6 +275,9 @@ uint64_t ebGetMaxExpireTime(ebuckets eb, EbucketsType *type, int accurate);
 
 uint64_t ebGetTotalItems(ebuckets eb, EbucketsType *type);
 
+typedef eItem (ebDefragFunction)(const eItem item);
+eItem ebDefragItem(ebuckets *eb, EbucketsType *type, eItem item, ebDefragFunction *fn);
+
```

Comment:
Please move it below the title `/* Item related API */`. Maybe place it after `uint64_t ebGetExpireTime()` since it is the least interesting a common user.

Reply:
done.